core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -1409,5 +1414,8 @@ class KafkaConfig(val props: java.util.Map[_, _], doLog: Boolean, dynamicConfigO
         s"${KafkaConfig.FailedAuthenticationDelayMsProp}=$failedAuthenticationDelayMs should always be less than" +
           s" ${KafkaConfig.ConnectionsMaxIdleMsProp}=$connectionsMaxIdleMs to prevent failed" +
           " authentication responses from timing out")
+
+    require(logCleanerMinCompactionLagMs <= 0 || logCleanerMaxCompactionLagMs <= 0 || logCleanerMinCompactionLagMs <= logCleanerMaxCompactionLagMs,
```

Comment:
I guess it should be `logCleanerMinCompactionLagMs >= 0 && logCleanerMaxCompactionLagMs >= logCleanerMinCompactionLagMs `

Reply:
there is a valid setting that logCleanerMaxCompactionLagMs = 0 (disabled) and logCleanerMinCompactionLagMs > 0.  I changed it a little bit to not allow negative value.